### PR TITLE
Alert-Button

### PR DESCRIPTION
**Pull request by ark2002**
- added the alert and button components
- added favicon to the badge component page
- linked the alert and button components on the components page

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  align-content: center;
+  padding: 1rem;
+  border-radius: 0.5rem;
+  border: 0.1rem solid;
+  width: 40%;
+  min-width: 30rem;
+}
+.alert__copy {
+  font-size: 1.6rem;
+  font-family: var(--secondary-font-family);
```
Comment (saifudinbadani) [inline components/alert/alert.css:16]: maybe rename it to a shorter variable name.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  border-color: var(--alert-primarycolor-success);
+}
+.alert--warning {
+  background-color: var(--alert-secondarycolor-warning);
+  color: var(--alert-primarycolor-warning);
+  border-color: var(--alert-primarycolor-warning);
+}
+.alert--error {
+  background-color: var(--alert-secondarycolor-error);
+  color: var(--alert-primarycolor-error);
```
Comment (saifudinbadani) [inline components/alert/alert.css:42]: if you are going to use this color as well, use a general name.

**Review by saifudinbadani — APPROVED** [review]
Looks good to merge

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
+.alert__demo {
+  flex-wrap: wrap;
+  gap: 10px;
+  padding: 2rem;
+}
+.alert {
+  align-content: center;
+  padding: 1rem;
+  border-radius: 0.5rem;
+  border: 0.1rem solid;
```
Comment (viru2001) [inline components/alert/alert.css:10]: for shorter lengths, you can use px instead of relative units

File: components/button/button.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+.floating__btn>.btn{
+    border:none;
+    border-radius:50%;
+    box-shadow: 0 0.3rem 0.6rem #00000029, 0 0.3rem 0.6rem #0000003b;;
+}
+.btn__icon--round>.btn{
+    border-radius:50%;
+    border:0.1rem solid #000000;
+    color: #000000;
+}
```
Comment (syed-shaida-hussain) [inline components/button/button.css:91]: you can use variable for color as you have uses variables everywhere.

File: components/alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+}
+.alert__title {
+  padding: 0.2rem 0;
+  
+}
+.alert--info {
+  background-color: var(--alert-secondarycolor-info);
+  color: var(--alert-primarycolor-info);
+  border-color: var(--alert-primarycolor-info);
+}
```
Comment (mohitdhatrak) [inline components/alert/alert.css:29]: maybe use a hyphen between like this primary-color

**Review by syed-shaida-hussain — CHANGES_REQUESTED** [review]

File: components/alert/alert.html [hunk 6]
```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="A design system for enterprise-level products. Clean HTML with component classes .Customizable and theme able."
+    />
```
Comment (mohitdhatrak) [inline components/alert/alert.html:10]: nice work adding this

File: components/button/button.html [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+      </button>
+      <button class="btn btn-color--tertiary btn-font--secondary">
+        tertiary
+      </button>
+    </div>
+    <p class="documentation">Outline Rectangular Buttons</p>
+    <div class="btn__outline">
+      <button class="btn btn-color--primary btn-font--primary">primary</button>
+      <button class="btn btn-color--secondary btn-font--primary">
+        secondary
```
Comment (mohitdhatrak) [inline components/button/button.html:96]: i saw live preview, it just has buttons no documentation (not just titles), please add that too before PRs. Mentioned by Kishan

File: components/alert/alert.html [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
```
Comment (viru2001) [inline components/alert/alert.html:14]: you can import this in your main CSS file so that you don't need to import it in HTML file

**Review by mohitdhatrak — COMMENTED** [review]
Few suggestions, otherwise looks good

File: components/button/button.css (hunk 4 above)
Comment (ark2002) [reply components/button/button.css:91]: this one missed my eye , fixed it rn you can have a look

File: components/alert/alert.css (hunk 5 above)
Comment (ark2002) [reply components/alert/alert.css:29]: the name for it was already pretty big so thought i could exclude the hyphen there .

File: components/alert/alert.css (hunk 2 above)
Comment (ark2002) [reply components/alert/alert.css:42]: the color was just specific for the components so used specific name . the ones which are gonna get reused are given generic names

File: components/button/button.css [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+.btn__outline > .btn-color--tertiary {
+  color: var(--tertiary-color);
+  border-color: var(--tertiary-color);
+}
+.btn--disabled>.btn{
+    opacity: 0.5;
+    cursor:not-allowed;
+}
+.btn--link>.btn{
+    color:#0000ff;
```
Comment (viru2001) [inline components/button/button.css]: you can declare variable for this color

File: components/button/button.css [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+}
+.floating__btn>.btn{
+    border:none;
+    border-radius:50%;
+    box-shadow: 0 0.3rem 0.6rem #00000029, 0 0.3rem 0.6rem #0000003b;;
+}
+.btn__icon--round>.btn{
+    border-radius:50%;
+    border:0.1rem solid #000000;
+    color: #000000;
```
Comment (viru2001) [inline components/button/button.css]: you should declare variable for black color

File: components/button/button.html (hunk 7 above)
Comment (ark2002) [reply components/button/button.html:96]: hey , currently lagging behind so my first priority currently is getting the components done first ill get done with the documentation as soon as i get done with these

**Review by viru2001 — CHANGES_REQUESTED** [review]
Good work. I have suggested some minor changes

File: components/alert/alert.css (hunk 3 above)
Comment (ark2002) [reply components/alert/alert.css:10]: i had used those in the earlier PR got reviewed that i should change everything to relative units so changed them to rem instead

File: components/alert/alert.html (hunk 8 above)
Comment (ark2002) [reply components/alert/alert.html:14]: i will look into it

File: components/button/button.css (hunk 10 above)
Comment (ark2002) [reply components/button/button.css]: changed it in the recent commit

File: components/button/button.css (hunk 9 above)
Comment (ark2002) [reply components/button/button.css]: changed it in the recent commit

File: components/alert/alert.css (hunk 3 above)
Comment (viru2001) [reply components/alert/alert.css:10]: ohhh ... you can use px for shorter lengths of 10px or lower than it

**Review by viru2001 — APPROVED** [review]

**Pull request merged**